### ISSUE TITLE
ForcedUserMode status not persisted in session

### ISSUE THREAD

**Issue by khteh** [zaproxy/zaproxy#5245]
**Describe the bug**
My site authentication works in the GUI after pressing the "Forced User Mode" to enable it. However, this toggle button status is not saved in the session which prevents me from running zap from command line successfully!

**To Reproduce**

**Expected behavior**
Able to run `zap.sh -cmd` using saved session successfully.

**Screenshots**

**Software versions**
 - ZAP: 2.7.0
 - Add-on: Name and version [e.g. Passive Scan Rules Alpha v19. https://github.com/zaproxy/zaproxy/wiki/FAQAddonVersions]
 - OS: Ubuntu 18.10
 - Java: 11.0.2
 - Browser: Chrome

**Errors from the zap.log file**
See https://github.com/zaproxy/zaproxy/wiki/FAQconfig for the location of the zap.log file for your OS.

**Additional context**

**Comment by psiinon** [comment]
Its worth noting that you can set forcedusermode via the API, which is probably more useful than allowing it to be set via the commandline.

**Comment by kingthorin** [comment]
Agreed, the API seems to have this covered. CLI shouldn’t be expected to do everything the ui/API do.

**Comment by kingthorin** [comment]
@Grunny's CLI wrapper might help you: 
https://github.com/Grunny/zap-cli
https://github.com/Grunny/zap-cli/issues/7#issuecomment-332825511

**Comment by kingthorin** [comment]
@Grunny's CLI wrapper might help you: 
https://github.com/Grunny/zap-cli
https://github.com/Grunny/zap-cli/issues/7#issuecomment-332825511

**Comment by khteh** [comment]
Start GUI > Configure > Run > Save Session > Close
Start GUI > Open saved session > Run > Fail!

Simple test case. Isn't this a bug!?! It only happens for NTLM/HTTP Auth. Basic Auth works fine. Root cause - ForcedUserMode is missing from the saved session!
